src/libraries/System.Reflection.Metadata/src/System/Reflection/Metadata/BlobWriter.cs

```diff
@@ -170,7 +170,7 @@ public void WriteBytes(BlobBuilder source)
                 Throw.ArgumentNull(nameof(source));
             }
 
-            source.WriteContentTo(ref this);
+            source!.WriteContentTo(ref this);
         }
 
         /// <exception cref="ArgumentNullException"><paramref name="source"/> is null.</exception>
```

Comment:
Cannot comment in the row 416, `public void WriteSerializedString(string? str)` should accept nullable string

also row 522 `public void WriteConstant(object? value)` value is nullable